backend/typescript/graphql/entityResolvers.ts

```diff
@@ -0,0 +1,44 @@
+import EntityService from "../services/implementations/EntityService";
+import { EntityRequestDTO } from "../services/interfaces/IEntityService";
+
+const entityService = new EntityService();
+
+const entityResolvers = {
+  Query: {
+    entity: async (_req: any, { id }: { id: string }) => {
+      console.log(id);
+      return entityService.getEntity(id);
+    },
+    entities: async () => {
+      return entityService.getEntities();
+    },
+  },
+  Mutation: {
+    createEntity: async (_req: any, obj: EntityRequestDTO) => {
```

Comment:
For consistency with the update mutation lets change the signature to `{ obj }: { obj: EntityRequestDTO }`, so the mutation call is now `createEntity(obj: {stringField: "test2", intField: 1, enumField: A, stringArrayField: ["A"], boolField: true})`